sub-logs/ricky.json

```diff
@@ -0,0 +1,21 @@
+{
+  "repo": "",
+
+  "owner": {
+    "username": "rickypurogu",
+    "email": "tanas.riccardo03@gmail.com"
+  },
+
+  "target": {
+    "CNAME": {
+            "name": "ricky",
+            "value": "rickypurogu.vercel.app"
+      },
+    "TXT": {
+       "name": "_vercel", 
```

Comment:
You cannot have a CNAME record with other records present. CNAME must exist by itself without anything else.

https://www.rfc-editor.org/rfc/rfc1034#section-3.6.2

Reply:
My bad, I thought it was necessary.